packages/domain/src/entities/__helpers__/mocks.ts

```diff
@@ -391,3 +396,21 @@ export function mockISignedVote(overrides?: Partial<ISignedVote>): ISignedVote {
 export function mockAppId(): AppId {
   return faker.commerce.productName() as AppId;
 }
+
+export function mockChargeFollowConfig(
+  overrides?: Partial<ChargeFollowConfig>,
+): ChargeFollowConfig {
+  return {
+    amount: mockUsdcAmount(42),
+    recipient: mockEvmAddress(),
+    ...overrides,
+    type: FollowPolicyType.CHARGE,
+  };
+}
+
+export function mockNoFeeFollowConfig(overrides?: Partial<NoFeeFollowConfig>): NoFeeFollowConfig {
+  return {
+    type: FollowPolicyType.ANYONE,
+    ...overrides,
+  };
+}
```

Comment:
Why this move?

`FollowPolicyConfig` are use case layer DTOs. I think their testing helper should live in the same layer, not in an inner layer (i.e. Entities)